### PR TITLE
Adds -r option to binary to pass the raw certificate

### PR DESCRIPTION
**Pull request by JoWilfrid**
### Original need

The original need is to have the ability not to store our certificates on the disk.
An additionnal need is to have custom loading process for the certificates in the app.

### Faced problem

The problem here is that the binary only takes a path to the certificate through the `-c` option.

### Proposed solution

The solution I want to propose here is to add a `-r` option to the binary so that we can pass the raw certificate.

### Let's discuss

I do not have the whole project context so maybe the way I did that is not the more efficient, I would be very pleased to have your feedbacks!

### CODE REVIEW

**Comment by JoWilfrid** [comment]
No one here?

File: bin/apn [hunk 1]
```diff
@@ -31,6 +31,7 @@ command :push do |c|
   c.option '-P', '--payload PAYLOAD', 'JSON payload for notifications'
   c.option '-e', '--environment ENV', [:production, :development], 'Environment to send push notification (production or development (default))'
   c.option '-c', '--certificate CERTIFICATE', 'Path to certificate (.pem) file'
+  c.option '-r', '--rawcert CERTIFICATE VALUE', 'Value of the certificate (.pem) file'
```
Comment (dankimio) [inline bin/apn]: Should we write the full parameter name like `raw-certificate`?

File: bin/apn [hunk 2]
```diff
@@ -126,6 +132,7 @@ command :feedback do |c|
 
   c.example 'description', 'apn feedback -c /path/to/apple_push_certificate.pem'
   c.option '-c', '--certificate CERTIFICATE', 'Path to certificate (.pem) file'
+  c.option '-r', '--rawcert CERTIFICATE VALUE', 'Value of the certificate (.pem) file'
```
Comment (dankimio) [inline bin/apn]: Same here.

Comment (dankimio) [reply bin/apn]: Perhaps we could add more description here? For example, how can one get value of the certificate or what would it look like.

File: bin/apn (hunk 1 above)
Comment (JoWilfrid) [reply bin/apn]: Yep, will do so!

File: bin/apn (hunk 2 above)
Comment (JoWilfrid) [reply bin/apn]: You mean something like `Actual content / value of the certificate (.pem) file`?

Comment (dankimio) [reply bin/apn]: Yes, something like that!

**Pull request merged**